homeassistant/components/nina/config_flow.py

```diff
@@ -101,6 +109,8 @@ async def async_step_user(
                         int, vol.Range(min=1, max=20)
                     ),
                     vol.Required(CONF_FILTER_CORONA, default=True): cv.boolean,
+                    vol.Optional(CONF_SINGLE_SENSOR, default=False): cv.boolean,
+                    vol.Optional(CONF_MULTIPLE_SENSOR, default=True): cv.boolean,
```

Comment:
There are non-critical/non-primary options and should therefore not be part of the configuration flow. Instead, these should be handled in an options flow.

Reply:
Then I would wait for #65890 and convert this PR to a draft